latest/ug/cluster-management/cost-monitoring-kubecost.adoc

```diff
@@ -18,104 +18,15 @@ Kubecost v2 introduces several major new features. <<kubecost-v2,Learn more abou
-
-+
-* You can remove Kubecost from your cluster with the following commands.
-+
-[source,bash,subs="verbatim,attributes"]
-----
-helm uninstall kubecost --namespace kubecost
-----
-+
-[source,bash,subs="verbatim,attributes"]
-----
-kubectl delete namespace kubecost
-----
-
-[#kubecost-dashboard]
-== Access Kubecost Dashboard
-. Make sure the required Pods are running.
-
-[source,bash,subs="verbatim,attributes"]
-----
-kubectl get pods --namespace kubecost
-----
-
-An example output is as follows.
-
-[source,bash,subs="verbatim,attributes"]
-----
-NAME                                          READY   STATUS    RESTARTS   AGE
-kubecost-cost-analyzer-b9788c99f-5vj5b        2/2     Running   0          3h27m
-kubecost-kube-state-metrics-99bb8c55b-bn2br   1/1     Running   0          3h27m
-kubecost-prometheus-server-7d9967bfc8-9c8p7   2/2     Running   0          3h27m
-----
-. On your device, enable port-forwarding to expose the Kubecost dashboard.
-+
-[source,bash,subs="verbatim,attributes"]
-----
-kubectl port-forward deployment/kubecost-cost-analyzer 9090 --namespace kubecost
-----
-+
-Alternatively, you can use the <<aws-load-balancer-controller,{aws} Load Balancer Controller>> to expose Kubecost and use Amazon Cognito for authentication, authorization, and user management. For more information, see link:containers/how-to-use-application-load-balancer-and-amazon-cognito-to-authenticate-users-for-your-kubernetes-web-apps[How to use Application Load Balancer and Amazon Cognito to authenticate users for your Kubernetes web apps,type="blog"].
-. On the same device that you completed the previous step on, open a web browser and enter the following address.
-+
-[source,bash,subs="verbatim,attributes"]
-----
-http://localhost:9090
-----
-+
-You see the Kubecost Overview page in your browser. It might take 5–10 minutes for Kubecost to gather metrics. You can see your Amazon EKS spend, including cumulative cluster costs, associated Kubernetes asset costs, and monthly aggregated spend.
-+
-image::images/kubecost.png[Kubecost dashboard,scaledwidth=100%]
-. To track costs at a cluster level, tag your Amazon EKS resources for billing. For more information, see <<tag-resources-for-billing>>.
+* If you are new to Amazon EKS we recommend that you use Helm for the installation because it simplifies the _Amazon EKS optimized Kubecost bundle_ installation. For more information, see <<cost-monitoring-kubecost-addons>>. 
+* For advanced users, you may configure your _Amazon EKS optimized Kubecost bundle_ with Helm. For more information, see <<cost-monitoring-kubecost-helm>>.
```

Comment:
To keep document style aligned as [how AWS-LBC present](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html#lbc-overview).